3.0 migration guide

mindreeper2420 opened the pull request:
This updates the Migrations Guides to include items for migrating base PatternFly from 2.x to 3.0 and Angular-PatternFly from 2.x to 3.0.

dlabrecq commented on source/get-started/patternfly-angular-migration-guide-3.0/index.md:
```diff
@@ -0,0 +1,84 @@
+---
```
Instead of having separate directories, each with their own index.md files, can we provide explicit file names? For example:

source/patternfly-migration-guides/angular-patternfly-migration-guide-3.0.md
source/patternfly-migration-guides/patternfly-migration-guide-2.0.md
source/patternfly-migration-guides/patternfly-migration-guide-3.0/index.md

Also note the "angular-patternfly" file name.

mindreeper2420 commented:
In addition to combining the files into one folder, the file name has been fixed for "angular-patternfly" to match the proper convention.

dlabrecq commented:
Not sure what happened, but there are 38 file changes and 4 commits now. Can you rebase and squash the commits?

dtaylor113 commented on source/get-started/patternfly-migration-guides/angular-patternfly-migration-guide-3.0.md:
```diff
@@ -0,0 +1,84 @@
+
+### Release Notes:
+
+  * Updated devDependency to PatternFly for 3.0.0
```
Do we want to mention the latest/current version?  patternfly 3.8.1?

dtaylor113 replied:
I think your right.  Typically, migration guides document a migration to a major release, so let's just keep it at 3.0.0

dtaylor113 commented:
I see 7 commits and 4 file changes

mindreeper2420 commented:
@dtaylor113 - I rebased and squashed, but it didn't appear to do anything with the commits. Fixed the files that were merged, but history is all jacked up now.

dlabrecq commented:
You could cherry pick your changes to a new branch, then force push -- using the same branch name. SourceTree makes cherry picking easy.
